ADAL/src/utils/ADClientCapabilitiesUtil.h

```diff
@@ -25,8 +25,6 @@
 
 @interface ADClientCapabilitiesUtil : NSObject
 
-+ (NSArray<NSString *> *)knownCapabilities:(NSArray<NSString *> *)allCapabilities;
-
 /*
```

Comment:
I like the change as we don't need to update library if server come up with a new capability :)

Reply:
👍